### PR TITLE
Feat issue#3 card

### LINKED ISSUE #3: First component!
Here we are!

Let's go to develop our first component!
The challenge is:

image
Moviee card!

Don't forget about your component typing and documentation! 

### Utils
- [moviee.zip](https://github.com/joao-leal12/Movie-/files/9892732/moviee.zip) - figma layout
- [logo.zip](https://github.com/joao-leal12/Movie-/files/9892913/logo.zip)

nice

### PR DESCRIPTION
**Pull request by joao-leal12**
resolve #3, Realizado a criação e documentação com o storybook do componente

### CODE REVIEW

File: src/components/ContainerMovies/ContainerMovies.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { ContainerMovies } from '.';
+import { Meta, StoryObj } from '@storybook/react';
+// eslint-disable-next-line @typescript-eslint/consistent-type-assertions
```
Comment (saleszera) [inline src/components/ContainerMovies/ContainerMovies.stories.tsx]: I suggest to disable this rule on `.eslintrc`

File: src/components/ContainerMovies/index.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { chakra } from '@chakra-ui/react';
+import { MoviesList } from '../MoviesList';
+export const ContainerMovies = (): JSX.Element => {
```
Comment (saleszera) [inline src/components/ContainerMovies/index.tsx]: Here you can omit the `JSX.Element`

File: src/components/MovieCard/MovieCard.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { MovieCard } from './index';
+import { Meta, StoryObj } from '@storybook/react';
+
+// eslint-disable-next-line @typescript-eslint/consistent-type-assertions
```
Comment (saleszera) [inline src/components/MovieCard/MovieCard.stories.tsx:4]: I suggest to disable this rule on `.eslintrc`

File: src/components/MovieCard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  Image,
+  Heading,
+  Text,
+  List,
+  ListItem,
+} from '@chakra-ui/react';
+import CardImage from '../../assets/card-image.png';
+import StarFull from '../../assets/starFull.png';
+import StarEmpty from '../../assets/starEmpty.png';
+export const MovieCard = (): JSX.Element => {
```
Comment (saleszera) [inline src/components/MovieCard/index.tsx]: you can omit `JSX.Element`

File: src/components/MoviesList/MoviesList.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,9 @@
+/* eslint-disable @typescript-eslint/consistent-type-assertions */
```
Comment (saleszera) [inline src/components/MoviesList/MoviesList.stories.tsx:1]: I suggest to disable this rule on .eslintrc

File: src/components/MovieCard/index.tsx [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+      bg="white"
+      p="2.4rem"
+      maxW="343px"
+      borderTopLeftRadius="10rem"
+      borderTopRightRadius="1.6rem"
+      borderBottomLeftRadius="1.6rem"
+      borderBottomRightRadius="10rem"
+    >
+      <Link display="block" _hover={{ textDecor: 'none' }}>
+        <Image src={CardImage} maxW="295px" />
```
Comment (saleszera) [inline src/components/MovieCard/index.tsx]: it's a best practice to use `alt` properties on ``

File: src/components/MovieCard/index.tsx [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+  WrapItem,
+  Link,
+  Image,
+  Heading,
+  Text,
+  List,
+  ListItem,
+} from '@chakra-ui/react';
+import CardImage from '../../assets/card-image.png';
+import StarFull from '../../assets/starFull.png';
```
Comment (saleszera) [inline src/components/MovieCard/index.tsx:11]: about stars, it should be an icon from [phosphor](https://phosphoricons.com/) and then it's better to use a svg icon instead `.png`

File: src/components/MovieCard/index.tsx [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+          mt="1.3rem"
+          whiteSpace="nowrap"
+        >
+          Home Alone 2
+        </Heading>
+        <Text fontSize="1.8rem" letterSpacing="0.16em">
+          Comedy
+        </Text>
+        <List display="flex">
+          <ListItem>
```
Comment (saleszera) [inline src/components/MovieCard/index.tsx:42]: bro, there is another way to render it for example:
```typescript
import { useMemo } from 'react';

import StarEmpty from '../../assets/starEmpty.png';
import StarFull from '../../assets/starFull.png';

interface IStars {
  rating: number;
}

const Stars = ({ rating }: IStars) => {
  const maxRating = 100;
  const maxStars = 5;

  const starsWillFilled = useMemo(
    () => Math.round((maxStars * rating) / maxRating),
    [maxStars, rating, maxRating]
  );
  const stars = new Array(maxStars).fill('');

  return (
    <List display="flex">
      {stars.map((_, index) => (
        <ListItem>
          <Image
            src={index < starsWillFilled ? StarFull : StarEmpty}
            alt={`start ${index < starsWillFilled ? 'filled' : 'unfilled'}`}
          />
        </ListItem>
      ))}
    </List>
  );
};
```

**Review by saleszera — CHANGES_REQUESTED** [review]

File: src/components/ContainerMovies/ContainerMovies.stories.tsx (hunk 1 above)
Comment (joao-leal12) [reply src/components/ContainerMovies/ContainerMovies.stories.tsx]: this is ok !

File: src/components/ContainerMovies/index.tsx (hunk 2 above)
Comment (joao-leal12) [reply src/components/ContainerMovies/index.tsx]: this is ok !

File: src/components/MovieCard/MovieCard.stories.tsx (hunk 3 above)
Comment (joao-leal12) [reply src/components/MovieCard/MovieCard.stories.tsx:4]: this is ok!

File: src/components/MovieCard/index.tsx (hunk 4 above)
Comment (joao-leal12) [reply src/components/MovieCard/index.tsx]: this is ok!

File: src/components/MovieCard/index.tsx (hunk 6 above)
Comment (joao-leal12) [reply src/components/MovieCard/index.tsx]: this is ok!

File: src/components/MovieCard/index.tsx (hunk 7 above)
Comment (joao-leal12) [reply src/components/MovieCard/index.tsx:11]: I will use it when fetching data dynamically!!

File: src/components/MovieCard/index.tsx (hunk 8 above)
Comment (joao-leal12) [reply src/components/MovieCard/index.tsx:42]: I will use it when fetching data dynamically

**Review by saleszera — APPROVED** [review]
Good job bro!!!

**Pull request merged**